packages/amplify-cli/src/__tests__/extensions/amplify-helpers/remove-resource.test.ts

```diff
@@ -156,7 +156,7 @@ describe('remove-resource', () => {
           },
           serviceSuffix: { Lambda: '(function)', LambdaLayer: '(layer)' },
         }),
-      ).rejects.toThrowError('An error occurred when removing the resources from the local directory');
+      ).rejects.toThrowError('Resource cannot be removed because it has a dependency on another resource');
```

Comment:
We should also assert on the type of the error since that's usually the culprit for these issues